### DISCUSSION TITLE
Fonctionnalités débloquées en fonction des niveaux

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#683] [Idées acceptées / déjà implémentées]
Pour les prochaines fonctionnalités, penser à les faire débloquer à partir d'un certain niveau, pour faire dans la continuité des fight et guild
Là, les niveaux ont une grosse utilité au début et au bout de 2 fonctionnalités pouf ça sert plus que pour les stats

**Comment by BastLast** [comment]
c'est déjà le cas actuellement et ca sera encore le cas dans le futur, je déplace ca dans idées acceptées et je lock vu que c'est déjà dans le bot
